spec/factories/state_file_archived_intakes.rb

```diff
@@ -13,6 +13,7 @@
 #  mailing_state           :string
 #  mailing_street          :string
 #  mailing_zip             :string
+#  permanently_locked_at   :datetime
```

Comment:
how come: `"failed_attempts", "locked_at"` are not represented in the annotations?

Reply:
yea not a huge problem